bindings/python/math/expose-rpy.cpp

```diff
@@ -22,35 +22,59 @@ namespace pinocchio
     {
       return pinocchio::rpy::matrixToRpy(R);
     }
-    
+
+    Eigen::Matrix3d rotate(const std::string axis, const double ang)
```

Comment:
Use ref here.

Reply:
And you can also take a char here, have you tried it?

Reply:
You are right about the ref. A char was my first choice. However, it seems that it was bound to Python strings of any length, which I found fishy and undesirable. Example: `rotate('xwhatever', pi)` would pass. A string allows me to throw an exception if the length is not equal to 1.

Reply:
Thanks for the corrections!